aviary/interface/methods_for_level2.py

```diff
@@ -381,14 +391,16 @@ def load_inputs(self, aviary_inputs, phase_info=None, engine_builders=None, meta
             aviary_inputs.set_val(Mission.Summary.GROSS_MASS,
                                   val=self.initialization_guesses['actual_takeoff_mass'], units='lbm')
             if 'target_range' in self.post_mission_info:
-                aviary_inputs.set_val(Mission.Design.RANGE, wrapped_convert_units(
+                aviary_inputs.set_val(Mission.Summary.RANGE, wrapped_convert_units(
```

Comment:
I think this is the fix for the multiengine tests regressions!

Reply:
Yeah you must set this to `target_range` because that is how far we are actually flying. Whereas design.Range is really just for sizing subsystems, weights, and max mission range.